xml-conduit/Text/XML/Stream/Parse.hs

```diff
@@ -978,6 +979,13 @@ many' :: MonadThrow m
       -> Consumer Event m [a]
 many' consumer = manyIgnore consumer ignoreAllTreesContent
 
+-- | Keep parsing elements as long as the parser returns 'Just', but throw the
+--   elements away without building an intermediate list.
+skipMany :: MonadThrow m
+         => Consumer Event m (Maybe a)
+         -> Consumer Event m ()
+skipMany consumer = manyIgnoreYield (return Nothing) ((() <$) <$> consumer)
```

Comment:
Yes, I think I initially tried `() <$` which doesn't map inside the Maybe and didn't think of using `void` when changing is, I've changed that now.